implementations.md

```diff
@@ -29,6 +29,7 @@ Here are a list of implementations that live in Fantasy Land:
 * [List in JS](https://github.com/PandaNoir/List-in-JS/) implements Setoid, Semigroup, Monoid, Functor, Applicative, Foldable, Traversable, Chain and Monad.
 * [Sanctuary](https://github.com/plaid/sanctuary) is a refuge from unsafe JavaScript. It provides FL-compatible Either and Maybe types.
 * [Fluture](https://github.com/Avaq/Fluture) is a high-performance monadic alternative to Promises.
+* [Ramda Adjunct](https://github.com/char0n/ramda-adjunct) is a community maintained extension of Ramda
```

Comment:
I think there should be a hyphen (`-`) between _community_ and _maintained_: a _community-maintained_ extension.